dkist/dataset/tiled_dataset.py

```diff
@@ -124,5 +126,29 @@ def shape(self):
         """
         return self._data.shape
 
+    @property
+    def tiles_shape(self):
+        """
+        The shape of each individual tile in the TiledDataset.
+        """
+        return [[tile.data.shape for tile in row] for row in self]
+
+    def slice_tiles(self, slice_):
```

Comment:
We should have this be a property so you can do `.slice_tiles[10:]` rather than `.slice_tiles(np.s_[10:])`